LogAndStream/python_scripts/Bluetooth commands/test_bt_cmds.py

```diff
@@ -946,47 +980,42 @@ def test_77_set_calib_dump(self):
                                     shimmer_comms_bluetooth.BtCmds.GET_CALIB_DUMP_COMMAND,
                                     shimmer_comms_bluetooth.BtCmds.RSP_CALIB_DUMP_COMMAND, 2)
 
-    def test_78_set_crc(self):
-        print("Test 78 - Set CRC Command")
-
-        if not self.shimmer.is_hardware_version_set():
-            self.test_04_get_shimmer_new_version(True)
-
+    def test_078_set_crc(self):
+        print("Test 078 - Set CRC Command")
```

Comment:
put back in:

         if not self.shimmer.is_hardware_version_set():
            self.test_04_get_shimmer_new_version(True)

as this test relies on it to determine num_inquiry_bytes